### PR TITLE
Refactor README.md and pull_request_template.md

### PR DESCRIPTION
**Pull request by CremaLuca**
## Premise
Too little information on the main README and too many useless details in the pull request template.
Also wanted the README to look cool because it's in our public profile and will probably end up in our curriculum.

#### `README.md`

**+** Getting started section containing:
 - Requirements
 - Installation
 - Database

**~** Added the secrets-volume method to the configurations

#### `pull_request_template.md`

Conventional commits are no use for a pull request. Also, there was too much to delete before you can start writing anything.

## Extra

Sadly `pull_request_template.md` changes will take effect only when we merge develop branch in the master branch.

Feel free to add anything if you wanted to.

## MayMay of the week
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,61 @@
...
 
-## config.json
+**Open Trading Research Infrastructure (OTRI)** is a research platform for financial computing focusing on data retrieval, validation, and analysis.
+
+## Getting started
+
+### Requirements
+
+- Any OS that supports Python 3.7
+- Python 3.7 and PIP
```
Comment (CremaLuca) [inline README.md:10]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/OTRI-Unipd/OTRI/pullRequest?prid=5985894)

**Comment by CremaLuca** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 11
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/OTRI-Unipd/OTRI/pullRequest?prid=5985894&bid=19628828)

File: README.md [hunk 2]
```diff
@@ -1,19 +1,61 @@
...
+$~: pip install -r requirements.txt
+```
+
+### Database
+
+Currently, only a PostgreSQL database is supported.
+The structure of the database is the following:
+
+- An `atoms` table that contains all retrieved data [ID : BigSerial, data_json : jsonb]
+- A `metadata` table that contains static metadata [ID : BigSerial, data_json : jsonb]
```
Comment (CremaLuca) [inline README.md:26]: Codacy found an issue: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/OTRI-Unipd/OTRI/pullRequest?prid=5985894)

File: README.md [hunk 3]
```diff
@@ -1,19 +1,61 @@
...
+
+To bootstrap scripts, you'll have to populate the `metadata` table with rows that contain the `ticker` key.
+> [!NOTE]
```
Comment (CremaLuca) [inline README.md:29]: Codacy found an issue: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/OTRI-Unipd/OTRI/pullRequest?prid=5985894)

File: README.md [hunk 4]
```diff
@@ -1,19 +1,61 @@
...
+$~: cd OTRI
+$~: pip install -r requirements.txt
+```
+
+### Database
+
+Currently, only a PostgreSQL database is supported.
+The structure of the database is the following:
+
+- An `atoms` table that contains all retrieved data [ID : BigSerial, data_json : jsonb]
```
Comment (CremaLuca) [inline README.md:25]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/OTRI-Unipd/OTRI/pullRequest?prid=5985894)

File: README.md [hunk 5]
```diff
@@ -1,19 +1,61 @@
...
+## Getting started
+
+### Requirements
+
+- Any OS that supports Python 3.7
+- Python 3.7 and PIP
+
+### Installation
+
+```bash
```
Comment (CremaLuca) [inline README.md:14]: Codacy found an issue: [[no-shell-dollars] Do not use dollar signs before shell commands](https://app.codacy.com/gh/OTRI-Unipd/OTRI/pullRequest?prid=5985894)

File: README.md (hunk 2 above)
Comment (CremaLuca) [inline README.md:26]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/OTRI-Unipd/OTRI/pullRequest?prid=5985894)

File: README.md [hunk 6]
```diff
@@ -1,19 +1,61 @@
...
-Open Trading Research Infrastructure.
 
-## config.json
+**Open Trading Research Infrastructure (OTRI)** is a research platform for financial computing focusing on data retrieval, validation, and analysis.
+
+## Getting started
+
+### Requirements
+
+- Any OS that supports Python 3.7
```
Comment (CremaLuca) [inline README.md:9]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/OTRI-Unipd/OTRI/pullRequest?prid=5985894)

File: README.md (hunk 3 above)
Comment (CremaLuca) [inline README.md:29]: Codacy found an issue: [[no-shortcut-reference-link] Use the trailing `[]` on reference links](https://app.codacy.com/gh/OTRI-Unipd/OTRI/pullRequest?prid=5985894)

File: README.md (hunk 2 above)
Comment (CremaLuca) [inline README.md:26]: Codacy found an issue: [[no-shortcut-reference-link] Use the trailing `[]` on reference links](https://app.codacy.com/gh/OTRI-Unipd/OTRI/pullRequest?prid=5985894)

File: README.md (hunk 4 above)
Comment (CremaLuca) [inline README.md:25]: Codacy found an issue: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/OTRI-Unipd/OTRI/pullRequest?prid=5985894)

Comment (CremaLuca) [inline README.md:25]: Codacy found an issue: [[no-shortcut-reference-link] Use the trailing `[]` on reference links](https://app.codacy.com/gh/OTRI-Unipd/OTRI/pullRequest?prid=5985894)

**Review by ricdezen — APPROVED** [review]
Apruvd. Solv konflikt and merg.

**Pull request closed without merging**